recipes/denoptim/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set version = "3.0.1" %}
+{% set sha256 = "22cfc7a343c72faf6a01194c8f2e29753f6680ef419646ef3bbc21ab9db0e3bf" %}
+{% set openjdk_version = "11.0.*" %}
+
+package:
+  name: denoptim
+  version: {{ version }}
+
+source:
+  url: https://github.com/denoptim-project/DENOPTIM/archive/refs/tags/v3.0.1-alpha.zip
+  sha256: {{ sha256 }}
```

```suggestion
{% set version = "3.1.6" %}
{% set openjdk_version = "11.0.*" %}

package:
  name: denoptim
  version: {{ version }}

source:
  url: https://github.com/denoptim-project/DENOPTIM/archive/refs/tags/v{{ version }}.tar.gz
  sha256: d8dca592d7f309cfdf9cc9cbc1a643f1d0998f09e5b05d993928474096b623c7
```
It's not necessary to use a template for something that is used exactly once. Also, we have a bot that updates the SHA automatically.